packages/core/docs/integrate/boilerplate.md

```diff
@@ -0,0 +1,47 @@
+# Using the integration boilerplate
```

Comment:
This duplicates our [Integration Guide](https://docs.vuestorefront.io/v2/integrate/integration-guide.html)

Reply:
It was removed from that part to the new one.